src/CrudServiceProvider.php

```diff
@@ -76,17 +77,22 @@ public function register()
             return new CRUD($app);
         });
 
+//        dd(ExcelServiceProvider::class);
+
         // register its dependencies
         $this->app->register(\Backpack\Base\BaseServiceProvider::class);
         $this->app->register(\Collective\Html\HtmlServiceProvider::class);
         $this->app->register(\Barryvdh\Elfinder\ElfinderServiceProvider::class);
         $this->app->register(\Intervention\Image\ImageServiceProvider::class);
 
+        $this->app->register(ExcelServiceProvider::class);
```

Comment:
Given the other registeres are using the fully qualified version (which I don't like but that's how things are), this one probably should too for consistency.